modules/products/bitbucket/locals.tf

```diff
@@ -11,6 +11,14 @@ locals {
     "mem" : var.bitbucket_configuration["mem"]
   }
 
+  admin_settings = length(kubernetes_secret.admin_secret) == 1 ? yamlencode({
+    bitbucket = {
+      sysadminCredentials = {
+        secretName = kubernetes_secret.admin_secret[0].metadata[0].name
+      }
+    }
+  }) : ""
```

Comment:
```suggestion
  }) : yamlencode({})
```

Reply:
Sorry, I saw this comment a bit late. I will apply this change in next PR.